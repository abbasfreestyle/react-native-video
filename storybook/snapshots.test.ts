import initStoryshots from '@storybook/addon-storyshots';

// https://github.com/storybookjs/storybook/issues/8083
jest.mock('global', () =>
  Object.assign(global, { window: { STORYBOOK_HOOKS_CONTEXT: '' } }),
);

initStoryshots();
