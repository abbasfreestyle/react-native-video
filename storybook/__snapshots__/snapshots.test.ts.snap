// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components Play Button 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      animatedStyle={
        Object {
          "value": Object {
            "transform": Array [
              Object {
                "scale": 1,
              },
            ],
          },
        }
      }
      collapsable={false}
      style={
        Object {
          "transform": Array [
            Object {
              "scale": 1,
            },
          ],
        }
      }
      testID="rn-video-play-button"
    >
      <View
        accessibilityRole="button"
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "height": 60,
            "justifyContent": "center",
            "width": 60,
          }
        }
      >
        <SvgMock
          height={50}
          testID="play-icon"
          width={50}
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots Components SVG Icons 1`] = `
<RCTScrollView>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-between",
          "paddingHorizontal": 50,
        }
      }
    >
      <Text>
        FastForward
      </Text>
      <SvgMock
        height={25}
        width={25}
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-between",
          "paddingHorizontal": 50,
        }
      }
    >
      <Text>
        Mute
      </Text>
      <SvgMock
        height={25}
        width={25}
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-between",
          "paddingHorizontal": 50,
        }
      }
    >
      <Text>
        Pause
      </Text>
      <SvgMock
        height={25}
        width={25}
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-between",
          "paddingHorizontal": 50,
        }
      }
    >
      <Text>
        Play
      </Text>
      <SvgMock
        height={25}
        width={25}
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-between",
          "paddingHorizontal": 50,
        }
      }
    >
      <Text>
        Rewind
      </Text>
      <SvgMock
        height={25}
        width={25}
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-between",
          "paddingHorizontal": 50,
        }
      }
    >
      <Text>
        Sound
      </Text>
      <SvgMock
        height={25}
        width={25}
      />
    </View>
  </View>
</RCTScrollView>
`;
